compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_block.rs

```diff
@@ -95,7 +105,10 @@ impl<'block, Registers: RegisterAllocator> BrilligBlock<'block, Registers> {
         &mut self,
         globals: &DataFlowGraph,
         used_globals: &HashSet<ValueId>,
-    ) {
+        hoisted_global_constants: &BTreeSet<(FieldElement, NumericType)>,
+    ) -> HashMap<(FieldElement, NumericType), BrilligVariable> {
+        let mut new_hoisted_constants = HashMap::default();
```

Comment:
Can you move this down to where it's being used?

Reply:
done